grafana-plugin/src/containers/OutgoingWebhookForm/OutgoingWebhookFormFields.tsx

```diff
@@ -45,345 +47,351 @@ const FORWARD_RADIO_OPTIONS = [
-              getSearchResult={grafanaTeamStore.getSearchResult}
-              displayField="name"
-              valueField="id"
-              placeholder="Choose (Optional)"
-              value={field.value}
-              onChange={field.onChange}
-            />
-          </Field>
-        )}
-      />
-      <Controller
-        name={WebhookFormFieldName.TriggerType}
-        control={control}
-        rules={{ required: 'Trigger Type is required' }}
-        render={({ field }) => (
-          <Field
-            label="Trigger Type"
-            description="The type of event which will cause this webhook to execute."
-            invalid={!!errors.trigger_type}
-            error={errors.trigger_type?.message}
-          >
-            <Select
-              placeholder="Choose (Required)"
-              value={field.value}
-              menuShouldPortal
-              options={WEBHOOK_TRIGGGER_TYPE_OPTIONS}
-              onChange={({ value }) => field.onChange(value)}
-            />
-          </Field>
-        )}
-      />
-      <Controller
-        name={WebhookFormFieldName.HttpMethod}
-        control={control}
-        rules={{ required: 'HTTP Method is required' }}
-        render={({ field }) => (
-          <Field label="HTTP Method" invalid={!!errors.http_method} error={errors.http_method?.message}>
-            <Select
-              placeholder="Choose (Required)"
-              value={field.value}
-              menuShouldPortal
-              options={HTTP_METHOD_OPTIONS}
-              onChange={({ value }) => field.onChange(value)}
-            />
-          </Field>
-        )}
-      />
-      <Controller
-        name={WebhookFormFieldName.IntegrationFilter}
-        control={control}
-        render={({ field }) => (
-          <Field
-            label="Integrations"
-            description="Integrations that this webhook applies to. If this is empty the webhook will execute for all integrations"
-            invalid={!!errors.integration_filter}
-            error={errors.integration_filter?.message}
-          >
-            <GSelect<ApiSchemas['AlertReceiveChannel']>
-              isMulti
-              placeholder="Choose (Optional)"
-              items={alertReceiveChannelStore.items}
-              fetchItemsFn={alertReceiveChannelStore.fetchItems}
-              fetchItemFn={alertReceiveChannelStore.fetchItemById}
-              getSearchResult={() => AlertReceiveChannelHelper.getSearchResult(alertReceiveChannelStore)}
-              displayField="verbal_name"
-              valueField="id"
-              getOptionLabel={(item: SelectableValue) => <Emoji text={item?.label || ''} />}
-              value={field.value}
-              onChange={field.onChange}
-            />
-          </Field>
-        )}
-      />
-      {hasLabelsFeature && (
+    if (loaderStore.isLoading(ActionKey.FETCH_INTEGRATION_CHANNELS)) {
+      return null;
+    }
```

Comment:
basically this is the main change here, along with wrapping this component in `observer`